service/gnomobiletypes/gnomobiletypes.go

```diff
@@ -58,6 +58,14 @@ type ListKeyInfoResponse struct {
 	Keys []*KeyInfo `json:"key_info" yaml:"key_info"`
 }
 
+type GetKeyInfoByNameRequest struct {
+	NameOrBech32 string `json:"name_or_bech32" yaml:"name_or_bech32"`
```

Comment:
It's confusing that the field is `NameOrBech32` and the method is `GetKeyInfoByNameRequest`, so it only by name or it's both name or bech32?

Reply:
As I replied to your other comment, maybe we should have all three:
GetKeyInfoByName(name), GetKeyInfoByAddress(bech32Addr) and GetKeyInfoByNameOrAddress(nameOrBech32). What do you think?

Reply:
Yes I agree.

Reply:
Hi @D4ryl00 . I force pushed a commit with these three functions. Can you review please?